composer.json

```diff
@@ -19,7 +19,8 @@
     },
     "autoload": {
         "psr-4": {
-            "EzSystems\\EzPlatformCodeStyle\\": "src/lib/"
+            "EzSystems\\EzPlatformCodeStyle\\": "src/bc/",
+            "Ibexa\\Platform\\CodeStyle\\": "src/lib/"
```

Comment:
AFAIR this was not supposed to be `Ibexa\\Platform` anymore

Reply:
Fixed in 5193217 and then in fe832eb.